packages/matter.js/src/protocol/ExchangeManager.ts

```diff
@@ -69,8 +77,16 @@ export class ExchangeManager<ContextT> {
     ) {}
 
     addTransportInterface(netInterface: TransportInterface) {
+        const udpInterface = netInterface instanceof UdpInterface;
         this.transportListeners.push(
             netInterface.onData((socket, data) => {
+                if (udpInterface && data.length > MAX_MESSAGE_SIZE) {
+                    logger.warn(
+                        `Ignoring UDP message with size ${data.length} from ${socket.name}, which is larger than the maximum allowed size of ${MAX_MESSAGE_SIZE}.`,
+                    );
+                    return;
```

Comment:
Do you think maybe this should be a warning too? Regardless of what the spec says, if someone sends the packet and we receive it sends like processing it is most likely what the user would want.

Reply:
Thats a good question :-)

On the other hand specs say:
> A message received over UDP that exceeds this message size limit SHALL NOT be processed.

I would respect hat for now until it first time happened and breaks for someone